docs/articles/documentation/using-testcafe/common-concepts/inject-scripts-into-tested-pages.md

```diff
@@ -0,0 +1,445 @@
+}
+{
+    "clientScripts": [ "<filePath>" | { "path": "<filePath>" } ]
+}
+```
+
+*Test API*
+
+```text
+fixture.clientScripts(filePath | { path: filePath })
+fixture.clientScripts(filePath | { path: filePath }, ...)
+fixture.clientScripts([ filePath | { path: filePath } ])
+```
+
+```text
+test.clientScripts(filePath | { path: filePath })
+test.clientScripts(filePath | { path: filePath }, ...)
+test.clientScripts([ filePath | { path: filePath } ])
+```
+
+Argument   | Type   | Description
+---------- | ------ | ---------------------------------------------------------------------------
+`filePath`, `filePath2`, `filePathN` | String | The path to the JavaScript file whose content should be injected.
+
+#### Relative Paths
+
+Relative paths resolve from the *current working directory* when you inject scripts in:
+
+* [command line interface](../command-line-interface.md#--cs-pathpath2---client-scripts-pathpath2)
+* [programming interface](../programming-interface/runner.md#clientscripts)
+* [configuration file](../configuration-file.md#clientscripts)
+
+When you use [test API methods](../../test-api/test-code-structure.md#inject-scripts-into-tested-pages), relative paths resolve from the *test file location*.
+
+**Examples**
+
+```sh
+testcafe chrome my-tests --cs assets/jquery.js
+```
+
+```js
+runner.clientScripts('assets/jquery.js');
+// or
+runner.clientScripts({ path: 'assets/jquery.js' });
+```
+
+```js
+{
+    "clientScripts": "assets/jquery.js",
+    // or
+    "clientScripts": { "path": "assets/jquery.js" }
+}
+```
+
+> You cannot combine the `path`, [module](#inject-a-module) and [content](#inject-script-code) properties.
+
+### Inject a Module
+
+Specify the Node.js module's name to inject its content into the tested pages. Use an object with the `module` property.
+
+*Programming interface*
+
+```text
+runner.clientScripts( { module: moduleName } )
+runner.clientScripts( { module: moduleName }, ... )
+runner.clientScripts([ { module: moduleName } ])
+```
+
+*Configuration file*
+
+```text
+{
+    "clientScripts": { "module": "<moduleName>" }
+}
+{
+    "clientScripts": [ { "module": "<moduleName>" } ]
+}
+```
+
+*Test API*
+
+```text
+fixture.clientScripts( { module: moduleName } )
+fixture.clientScripts( { module: moduleName }, ... )
+fixture.clientScripts([ { module: moduleName } ])
+```
+
+```text
+test.clientScripts( { module: moduleName } )
+test.clientScripts( { module: moduleName }, ... )
+test.clientScripts([ { module: moduleName } ])
+```
+
+Argument  | Type   | Description
+--------- | ------ | ----------------
+`moduleName`  | String | The module name.
+
+TestCafe searches for the module's entry point with Node.js mechanisms and injects its content into the tested page.
+
+Note that the browser must be able to execute the injected module. For example, modules that implement the [UMD](https://github.com/umdjs/umd) API can run in most modern browsers.
+
+> If the injected module has dependencies, you should ensure that:
```

Comment:
If the injected module has dependencies, ensure that the dependencies can be loaded as global variables and these variables are initialized in the page's code.

Reply:
I'm not sure why you use a list here.